.github/workflows/ci.yml

```
@@ -0,0 +1,66 @@
+name: Cross-Platform CI
```

```suggestion
name: Sanity checks
```

or Tokyo Nights, or something less boring?